Projeto-Stream/src/views/Home/homeDetail.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+    <div :style="{ backgroundImage: `linear-gradient(to right, black, rgba(0, 0, 0, 0.4)), url(${backgroundUrl})` }"
```

Comment:
Replace to tag main for better semantic. 
Here some links about semantic HTML tags: 
[https://web.dev/learn/html/semantic-html?hl=pt-br](https://web.dev/learn/html/semantic-html?hl=pt-br)
[https://www.w3schools.com/html/html5_semantic_elements.asp](https://www.w3schools.com/html/html5_semantic_elements.asp)